modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/declaration/PeerDeclarationHealthCheckDecision.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.sdk.infrastructure.healthcheck.declaration
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.domain.healthcheck.consensus.types.HealthCheckConsensusDecision
+
+final case class PositiveOutcome(peer: PeerId) extends HealthCheckConsensusDecision
```

Comment:
I think that the `peer` argument is never used. If that's the case, it could be a `case object` as well.